File: Node/Euler001.js
#!/usr/bin/env node

function main() {
  s = 0
  for (let i = 3; i < 1000; i +=3 ) {
    s += i
  }

  for (let i = 5; i < 1000; i += 5) {
    if (i % 3 == 0) {
      continue
    }
    s += i
  }

  console.log(s)
}


main()

File: Swift/Euler001.swift
#!/usr/bin/env swift
import Foundation

func main() {
  var sum: Int = stride(from: 3, to: 1000, by: 3).reduce(0, +)

  sum += stride(from: 5, to: 1000, by: 5).filter { x in x % 3 != 0 }.reduce(0, +)

  print(sum)
}

main()
